src/main/java/org/jenkinsci/test/acceptance/junit/JenkinsAcceptanceTestRule.java

```diff
@@ -38,6 +40,18 @@
  * @author Kohsuke Kawaguchi
  */
 public class JenkinsAcceptanceTestRule implements MethodRule { // TODO should use TestRule instead
+
+    private boolean enableInstallWizard;
```

Comment:
Committed a change with a new annotation @WithInstallWizard, now that looks a lot better. Thanks for the suggestion @olivergondza